js/app/DoCommand.html

```diff
@@ -76,6 +76,12 @@
       displayForm(form);
       break;
     }
+    case 'copyItems': {
+      const items = JSON.parse(app.pages.form.elements.form.items.value);
+      items.forEach(utility.stripItem);
```

Comment:
The thing I'm suggesting here does not exist, but I'm thinking maybe it should.  More explanation to come in the main thread.
```suggestion
      items.forEach(item => item.strip());
```

Reply:
or
```suggestion
      items.forEach(utility.item.strip);
```